types/pino-http/index.d.ts

```diff
@@ -39,4 +39,7 @@ declare module 'http' {
     interface IncomingMessage {
         log: Logger;
     }
+    interface ServerResponse {
+        err: Error;
```

Comment:
Should this be optional?

Reply:
Good point, yep - I've updated that. Thanks!